src/scss/columnFilter.scss

```
@@ -108,28 +108,46 @@
       }
     }
 
-    .column-filter-checkbox-wrapper {
-      padding: 0 10px;
-      input {
-        width: 30px;
+    .draggable {
+      display: flex;
+      align-items: center;
+      &:nth-child(odd){
```

Hier fehlt an mehreren Stellen Leerzeichen von Curly Brace
```suggestion
      &:nth-child(odd) {
```